funnel/models/event.py

```diff
@@ -137,11 +232,39 @@ class SyncTicket(BaseMixin, db.Model):
     participant_id = db.Column(None, db.ForeignKey('participant.id'), nullable=False)
     participant = db.relationship(Participant, primaryjoin=participant_id == Participant.id,
         backref=db.backref('sync_tickets', cascade="all, delete-orphan"))
-    proposal_space_id = db.Column(db.Integer, db.ForeignKey('proposal_space.id'), nullable=False)
-    proposal_space = db.relationship(ProposalSpace,
-        backref=db.backref('sync_tickets', cascade='all, delete-orphan'))
     ticket_client_id = db.Column(db.Integer, db.ForeignKey('ticket_client.id'), nullable=False)
     ticket_client = db.relationship(TicketClient,
         backref=db.backref('sync_tickets', cascade='all, delete-orphan'))
+    __table_args__ = (db.UniqueConstraint('ticket_client_id', 'order_no', 'ticket_no'),)
+
+    previous_participant = None
```

Comment:
A class-level variable? Every time you set it, you'll be editing the class, not the instance.